contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -143,6 +146,12 @@ message ApiAuthenticationConfig {
   bool force_disable = 1;
 }
 
+// Server config for API Authorization via Firebase Rules
+message ApiCheckSecurityRulesConfig {
+  // Allows to disable the API authorization
+  bool force_disable = 1;
+}
```

Comment:
This can be enabled via service config. But the service config changes are not done yet. Tao is working on those changes.